comp/dogstatsd/server/server.go

```diff
@@ -373,8 +380,20 @@ func (s *server) start(context.Context) error {
 		}
 	}
 
-	if len(socketPath) > 0 {
-		unixListener, err := listeners.NewUDSDatagramListener(packetsChannel, sharedPacketPoolManager, sharedUDSOobPoolManager, s.config, s.tCapture, s.wmeta, s.pidMap)
+	// Listen on the default socket (if /var/run/datadog/ exists)
+	if _, err := os.Stat(filepath.Dir(defaultSocket)); !os.IsNotExist(err) {
+		unixListener, err := listeners.NewUDSDatagramListener(packetsChannel, sharedPacketPoolManager, sharedUDSOobPoolManager, s.config, defaultSocket, s.tCapture, s.wmeta, s.pidMap)
+		if err != nil {
+			s.log.Errorf("Can't init listener: %s", err.Error())
```

Comment:
Would it make sense to specify what kind of listener and which path we tried to create?